docs/modules/usage/runtimes.md

```diff
@@ -117,3 +120,58 @@ bash -i <(curl -sL https://get.daytona.io/openhands)
 Once executed, OpenHands should be running locally and ready for use.
 
 For more details and manual initialization, view the entire [README.md](https://github.com/All-Hands-AI/OpenHands/blob/main/openhands/runtime/impl/daytona/README.md)
+
+## Local Runtime
+
+The Local Runtime allows the OpenHands agent to execute actions directly on your local machine without using Docker. This runtime is primarily intended for controlled environments like CI pipelines or testing scenarios where Docker is not available.
+
+:::caution
+**Security Warning**: The Local Runtime runs without any sandbox isolation. The agent can directly access and modify files on your machine. Only use this runtime in controlled environments or when you fully understand the security implications.
+:::
+
+### Prerequisites
+
+Before using the Local Runtime, ensure you have the following dependencies installed:
+
+1. A working installation of OpenHands following the [development setup instructions](https://github.com/All-Hands-AI/OpenHands/blob/main/Development.md)
+2. Jupyter installed in your Poetry environment
+3. libtmux installed and tmux available on your system
+4. Browser dependencies for the browser environment
```

Comment:
```suggestion
2. tmux available on your system
```

For jupyter and browser, `make build` should be able to install them automatically